admin_guide/preventing_node_placement.adoc

```diff
@@ -0,0 +1,101 @@
+= Preventing Specific Node Placement
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+As a cluster administrator, you can prevent users from targeting specific nodes
+when scheduling pods using labels and node selectors, as well as the node
+blacklist setting.  
+
+== Constraining Pod Placement
+
+Adding a label to the `nodeSelectorLabelBlacklist` section of a master
```

Comment:
This is incomplete and ambiguous; maybe it could be better worded like:

The `nodeSelectorLabelBlacklist` configuration field allows an administrator to control which labels certain users are able to specify in new Pod spec `nodeSelector` fields. Users, service accounts and members of groups which have the `pods/binding` permission can specify any `nodeSelector`; those without the `pods/binding` permission are prohibited from setting a `nodeSelector` for any label that appears in `nodeSelectorLabelBlacklist`.